testdata/policies/repos.yaml

```diff
@@ -15,7 +15,7 @@ policy:
       exposeports: "8080"
       branches:
           cgo: true
-          goversion: 1.15
+          goversion: 1.16
```

Comment:
This will apply to all the branches that doesn't have an explicit `goversion` key. I  believe the 1.16 changes are only for `master` and `release-4`. Branch specific policy take precedence over the common policy. So leaving this as it is, and setting `goversion` to `1.16` for `master` and `release-4` would be better I think.